corehq/apps/app_manager/tasks.py

```diff
@@ -1,9 +1,23 @@
 from celery.task import task
 from corehq.apps.users.models import CommCareUser
+from corehq.util.decorators import serial_task
 
 
 @task(queue='background_queue', ignore_result=True)
 def create_user_cases(domain_name):
     from corehq.apps.callcenter.utils import sync_usercase
     for user in CommCareUser.by_domain(domain_name):
         sync_usercase(user)
+
+
+@serial_task('{app._id}', max_retries=0)
```

Comment:
looks like the default timeout on the lock is 5 minutes - do we anticipate making the build to ever take longer than 5 minutes?